packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx

```diff
@@ -196,3 +197,7 @@ export function FloatingAddMenu<Action>({
         </FloatingContainer.Container>
     );
 }
+
+function hasOnlySpaces(text: string) {
```

Comment:
@e1himself @aspirisen If you have better place for that let me know :)

Reply:
Maybe put it here packages/slate-editor/src/lib ?

Reply:
@aspirisen hmm, let's leave as it is, if we would need in another place we then move it :)